components/cluster/service-hacluster-remote/files/svc-hacluster-remote

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/bash
+# Start/stop HACluster service
+#
+. /lib/svc/share/smf_include.sh
+
+#==#==# Pacemaker Remote
+# Use a custom directory for finding the authkey.
+PCMK_authkey_location=/etc/pacemaker/authkey
+#
+# Specify a custom port for Pacemaker Remote connections
+PCMK_remote_port=3121
+
+## Tracing with ebug version
+# PCMK_trace_files=1
+# PCMK_trace_functions=1
+# PCMK_trace_formats=1
+# PCMK_trace_tags=1
+
+export PCMK_ipc_type=socket
+export PCMK_logfile=/var/log/hacluster/pacemaker.log
+
+export CLUSTER_USER=hacluster
+export CLUSTER_GROUP=haclient
+export PATH=/usr/gnu/bin:${PATH}
+MYNAME=hacluster
```

Comment:
Why not also export these?

Reply:
This and the following variables are only used in the shell script and not by any subprocess